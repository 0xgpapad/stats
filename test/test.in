100 mon
150 mon
this -100 day
this 200 day
this300 day
100 200
200 300
Ending in dot 100.
Ending in dot 200.
Startint 100
Startint 200.2
Startint 300
Startfloat 100.1
Startfloat 200
Startfloat 300.1
Floating 100 1.5
Floating 200 2.5
Same number 100 equals
Same number 100 equals
