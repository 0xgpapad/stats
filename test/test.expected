100-150(125+/-25) mon
this -100-300(133.333+/-1.7e+02) day
100-200(150+/-50) 200-300(250+/-50)
Ending in dot 100-200(150+/-50).
Startint 100.000000-300.000000(200.067+/-82)
Startfloat 100.100000-300.100000(200.067+/-82)
Floating 100-200(150+/-50) 1.500000-2.500000(2+/-0.5)
Same number 100 equals
